tests/test_pacing_future.py

```diff
@@ -0,0 +1,64 @@
+"""This is a duplicate of test_pacing but uses future annotations"""
```

Comment:
Well now this docstring needs a small update.

But we can leave that for when i go through all the docstrings after the `ruff` stuff. 😩

Reply:
hehe.. I used ruff in it's current configuration and it didn't complain.  
Chop chop!